src/token.cairo

```diff
@@ -1,13 +1,21 @@
 // This is a freely tradable ERC20 token.
-// TODO upgradability
 
 #[starknet::contract]
 mod FloatingToken {
```

Comment:
Isn't this token missing mint and burn functions?

Reply:
The CARM token has fixed supply, so no mints/ burns

Reply:
In function `unstake_airdrop` in `staking.cairo` the `mint` function is called on floating_token

Reply:
fixed thx